### DISCUSSION TITLE
Time Format without AM and PM

### DISCUSSION THREAD

**Discussion by gradaom** [moment/moment#6415] [Q&A] [answered]
using `moment().format('LT')` , I can get result like 6:02 PM

I would like to get th result as 18:02 without setting am/pm

**Comment by antarilos** [comment, marked as answer]
It seems that you should use `moment().format('H:mm')`. (https://momentjs.com/docs/#/displaying/format/)
```js
const date = moment();
console.log(date.format('LT')); // 9:47 PM
console.log(date.format('H:mm')); // 21:47
console.log(date.format('HH:mm')); // 21:47. Just in case you need to precede a '0' when is one digit hour: 04:21
```

**Comment by gradaom** [comment]
hmm, then . @antarilos  , could you please add that precede `0` there when is one digit hour? Most of time , that's used more than not including

**Comment by antarilos** [comment]
That is just what is done using `date.format('HH:mm')`.
